src/Shared/EventArgsFormatting.cs

```diff
@@ -257,11 +258,19 @@ int threadId
             {
                 format.Append("{6}");
             }
-
+;
             // If the project file was specified, tack that onto the very end.
+            // Check for additional properties that should be outputted as well
             if (projectFile != null && !String.Equals(projectFile, file))
             {
-                format.Append(" [{10}]");
+                if (LogOutputProperties != null && LogOutputProperties.Length > 0)
+                {
+                    format.Append(" [{10}> ").Append(LogOutputProperties).Append("]");
```

Comment:
It feels a bit weird to actually expand something here when everything else is just creating holes for the final `Format` call. Is there an advantage to doing it this way?

Reply:
No, I can change it to be more consistent. My bad.